### DISCUSSION TITLE
Anyone using dataframe-jdbc from notebooks?

### DISCUSSION THREAD

**Discussion by apatrida** [Kotlin/dataframe#486] [Q&A]
I tried a basic notebook to try `DataFrame.readSqlTable(...)` and it does not work within Intellij Kotlin notebooks.

```
%useLatestDescriptors
%use dataframe
%use mysql

import org.jetbrains.kotlinx.dataframe.io.DatabaseConfiguration
import org.jetbrains.kotlinx.dataframe.io.readSqlTable

val dbHost = "..."
val dbPort = 3306
val dbName = "..."

val url = "jdbc:mysql://$dbHost:$dbPort/$dbName"
val username = "..."
val password = "..."

val dbConfig = DatabaseConfiguration(url, username, password)

val df = DataFrame.readSqlTable(dbConfig, "group_settings_name", 100)
df.print()
```

results in errors 

```
Line_57.jupyter.kts (12:51 - 72) Unresolved reference: DatabaseConfiguration
Line_57.jupyter.kts (14:20 - 32) Unresolved reference: readSqlTable
```

Anyone else have any luck?

**Comment by Jolanrensen** [comment]
The %use magics are supposed to be in a different cell. If yours are not, that should help :).

I also noticed on my machine, playing around with JDBC, thanks to incompatibilities with notebooks and the classpath JDBC queries, I need the "Select Project Libraries to add to the Notebook Classpath" setting and the project next to my notebook needs to depend on the JDBC library.
